src/api/spec/features/webui/maintenance_workflow_spec.rb

```diff
@@ -54,6 +54,17 @@
     click_button('Ok')
     expect(page).to have_css("#flash-messages", text: "Created maintenance incident request")
 
+    # Check that sending maintenance updates adds the source revision
+    new_bs_request_action = BsRequestAction.where(
+      type:                  "maintenance_incident",
+      target_project:        maintenance_project.name,
+      target_releaseproject: update_project.name,
+      source_project:        "#{user.home_project}:branches:#{update_project.name}",
+      source_package:        package.name
+    )
+    expect(new_bs_request_action).to exist
```

Comment:
To give some background, we added this when we thought we need to call ``` bs_request.bs_request_actions.first.source_rev ```.

You can drop that line and change the following one to ``` expect(new_bs_request_action.pluck(:source_rev).first).not_to be nil ```